### ISSUE TITLE
Failure in Mono run of System.Runtime.Serialization.Formatters.Tests

### ISSUE THREAD

**Issue by jkoritzinsky** [dotnet/runtime#89661]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=355524
Build error leg or test failing: System.Runtime.Serialization.Formatters.Tests.WorkItemExecution
Pull request: https://github.com/dotnet/runtime/pull/89643

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-section).

```json
{
  "ErrorMessage": "Microsoft.NETCore.App/8.0.0/libcoreclr.dylib : marker_idle_func",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=355524
**Error message validated:** `Microsoft.NETCore.App/8.0.0/libcoreclr.dylib : marker_idle_func`
**Result validation: :white_check_mark:** Known issue matched with the provided build.
**Validation performed at:** 7/29/2023 1:17:35 AM UTC

### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[355524](https://dev.azure.com/dnceng-public/public/_build/results?buildId=355524)|dotnet/runtime|[System.Runtime.Serialization.Formatters.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=355524&view=ms.vss-test-web.build-test-results-tab&runId=7477908&resultId=204443)|dotnet/runtime#89643|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|0|0|1|

**Comment by ivanpovazan** [comment]
@jkoritzinsky have you tried rerunning the failing test?

The build provided in this issue got removed.
I have inspected couple of runs from the past few days and the test in question passes on reported configuration:
- Build 1: https://dev.azure.com/dnceng-public/public/_build/results?buildId=372922&view=logs&j=ab5d32de-6d19-5068-e03b-b69cf4db6cd5&t=eb2a8345-733e-5823-7aa4-11fb75e4a1ff
    Test output: https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-90253-merge-7eb8d7482620432f90/System.Runtime.Serialization.Formatters.Tests/1/console.de54a312.log?helixlogtype=result
- Build 2: https://dev.azure.com/dnceng-public/public/_build/results?buildId=376832&view=logs&j=ab5d32de-6d19-5068-e03b-b69cf4db6cd5&t=eb2a8345-733e-5823-7aa4-11fb75e4a1ff
    Test output: https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-90695-merge-611466d9d9b34b0797/System.Runtime.Serialization.Formatters.Tests/1/console.f7f8da6f.log?helixlogtype=result
    
Additionally, the Build analysis service only picked 1 occurrence this month, so I believe this either got fixed, or was a flaky test. I suggest we close this issue.
